From b75c8a37f14a2a6e6e245284a565747a3a9fba7c Mon Sep 17 00:00:00 2001
From: Dmitry Ornatsky <cdo021c@motorola.com>
Date: Wed, 11 Nov 2015 08:57:47 -0600
Subject: [PATCH 463/959] IKKRNBSP-3500 arm/dt: 8952: change utag device node
 name

Change utag node name used for sysfs interface.

Change-Id: I7edc4c40864c411c3bcf7c2fa966549caade9d2f
Signed-off-by: Dmitry Ornatsky <cdo021c@motorola.com>
Reviewed-on: http://gerrit.mot.com/798928
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Igor Kovalenko <igork@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 Documentation/devicetree/bindings/misc/utags.txt | 11 +++++++++--
 arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi  |  2 +-
 2 files changed, 10 insertions(+), 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/misc/utags.txt b/Documentation/devicetree/bindings/misc/utags.txt
index 427da504d5b..2e93d1a7bf4 100644
--- a/Documentation/devicetree/bindings/misc/utags.txt
+++ b/Documentation/devicetree/bindings/misc/utags.txt
@@ -1,4 +1,5 @@
 MMI utags driver provides procfs access to the utags partition.
+Use unique names for device nodes: /sys/devices/utags etc.
 
 Required properties:
 - compatible		: string should be "mmi,utags"
@@ -9,10 +10,16 @@ Optional properties:
 
 Example:
 
-utags@1 {
+utags {
 	compatible = "mmi,utags";
 	status = "okay";
-	mmi,dir-name = "hwid";
 	mmi,utags-main = "/dev/block/bootdevice/by-name/utags";
 	mmi,utags-backup = "/dev/block/bootdevice/by-name/utagsBackup";
 };
+
+hw {
+	compatible = "mmi,utags";
+	status = "okay";
+	mmi,dir-name = "hw";
+	mmi,utags-main = "/dev/block/bootdevice/by-name/hw";
+};
diff --git a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
index 8416ff64a33..7caa3f6a9fe 100644
--- a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
@@ -55,7 +55,7 @@
 
 	};
 
-	utags@1 {
+	utags {
 		compatible = "mmi,utags";
 		mmi,main-utags = "/dev/block/bootdevice/by-name/utags";
 		mmi,backup-utags = "/dev/block/bootdevice/by-name/utagsBackup";
-- 
2.11.0

